### PR TITLE
Update getting started tour for binder

### LINKED ISSUE #10: Add Hello World pipeline tutorial to Binder getting started tour
Follow-up to #9. If a user launches binder the pipelines tutorial artifacts should be included by default, eliminating the need to download them from https://github.com/elyra-ai/examples/.

### LINKED ISSUE #19: Elyra binder getting starting document is out of date
In a couple of the screenshots in the `getting_started.md` file of Elyra, the screenshots / instructions are out of date. (below is one example - doesn't include adding / editing of code snippets)
image

### PR DESCRIPTION
**Pull request by ptitzler**
This PR:
- Adds a `README_1st` to get the user's attention
- Adds git and pipelines features 
- Closes #10 
- Closes #19 

### Use temporary hosted instance

Temporary preview URL: https://mybinder.org/v2/gh/ptitzler/elyra/do-not-merge-i826?urlpath=lab%2Ftree%2Fbinder-demo

### Locally

```
$ pip install jupyter-repo2docker
$ jupyter-repo2docker --ref do-not-merge-i826 https://github.com/ptitzler/elyra
```

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by ptitzler** [comment]
Couple of notes:
 - To avoid excessive duplication of content (which requires periodic refreshes as new pipeline features are released) the getting started tour requires the user to use the `git` feature to clone the examples repository (how meta) and follow the instructions in the hello world directory.

**Comment by ptitzler** [comment]
Found a binder specific issue that prohibits us from using the git feature. Might need to change the related content if I can't find a workaround.

**Comment by ptitzler** [comment]
This PR requires https://github.com/elyra-ai/elyra/pull/963 to be merged first.

File: binder/getting-started/getting_started.md [hunk 1]
```diff
@@ -17,13 +17,42 @@ limitations under the License.
 -->
 # Elyra quickstart
 
-This short interactive quickstart provides an introduction to Elyra - an AI centric extension for JupyterLab. Note that this tour does not include features requiring software that is not bundled with Elyra or JupyterLab, such as the Notebook pipeline editor, which requires a KubeFlow Pipelines installation. 
+This short interactive quickstart provides an introduction to Elyra - aset of AI centric extensions for JupyterLab. Note that this tour does not include features requiring software that is not bundled with Elyra or JupyterLab, such as Kubeflow Pipelines.
```
Comment (lresende) [inline binder/getting-started/getting_started.md:20]:
```suggestion
This short interactive quickstart provides an introduction to Elyra - a set of AI centric extensions for JupyterLab. Note that this tour does not include features requiring software that is not bundled with Elyra or JupyterLab, such as Kubeflow Pipelines.
```

**Review by lresende — APPROVED** [review]

**Pull request merged**